TicTacToeApp/ViewController.swift

```diff
@@ -32,11 +30,43 @@ public class ViewController: UIViewController {
     game.makeMove(button.tag)
     updateGameInformationDisplay()
 
+    if (!resetButton.enabled) {
```

Comment:
That logic no longer exists, but it was intended to make sure the reset button was enabled after the first move had been made. Not the best solution, I'll admit.